tests/foreman/ui/test_registration.py

```diff
@@ -303,24 +305,25 @@ def test_global_registration_form_populate(
         content_view=module_promoted_cv,
         group_parameters_attributes=[group_params],
     ).create()
+    target_sat.api.HostGroup(name=hg_nested_name, parent=parent_hg).create()
     new_org = target_sat.api.Organization().create()
     new_ak = target_sat.api.ActivationKey(organization=new_org).create()
     with session:
         session.hostgroup.update(
-            hg_name,
+            f'{hg_name}/{hg_nested_name}',
```

Comment:
Just one question here, do you think will it make sense to cover both parent and nested hostgroups here with for-loop here for this specific test?

Reply:
not convinced it is likely that data from parent hg would populate correctly when using child hg and at the same time not populate correctly when using parent directly. I'd assume parent is covered trough nesting if you know what I mean